docs/source/examples/checks/vision/performance/performance_report.ipynb

```diff
@@ -5,48 +5,52 @@
    "id": "e767e792",
```

Comment:
<p>As far as I know, average precision is per class, mean average precision is overall. So here it's Average Precision per class.</p><p>Also, it's done for a single IoU threshold that is 0.5 no?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/844/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>